js/commonJs.js

```diff
@@ -0,0 +1,462 @@
+document.addEventListener("DOMContentLoaded", function (e) {
```

Comment:
It's tricky, the goal to separate them was to keep the file you update regurlarly when doing implem of algolia really minimal.
Most of the code is not really understandable by everyone, I would keep it separate